docs/outputs.rst

```diff
@@ -6,118 +6,136 @@ Outputs of tedana
-Filename                  Content
-======================    =====================================================
-t2svG.nii.gz              Full T2* map/time series.
-                          Values are in seconds.
-                          The difference between the limited and full maps is
-                          that, for voxels affected by dropout where only one
-                          echo contains good data, the full map uses the
-                          single echo's value while the limited map has a NaN.
-                          Only used for optimal combination.
-s0vG.nii.gz               Full S0 map/time series. Only used for optimal
-                          combination.
-hik_ts_e[echo].nii.gz     High-Kappa time series for echo number ``echo``
-midk_ts_e[echo].nii.gz    Mid-Kappa time series for echo number ``echo``
-lowk_ts_e[echo].nii.gz    Low-Kappa time series for echo number ``echo``
-dn_ts_e[echo].nii.gz      Denoised time series for echo number ``echo``
-======================    =====================================================
+==============================================================  =====================================================
+Filename                                                        Content
+==============================================================  =====================================================
+desc-full_T2starmap.nii.gz                                      Full T2* map/time series.
+                                                                Values are in seconds.
+                                                                The difference between the limited and full maps is
+                                                                that, for voxels affected by dropout where only one
+                                                                echo contains good data, the full map uses the
+                                                                single echo's value while the limited map has a NaN.
+                                                                Only used for optimal combination.
+desc-full_S0map.nii.gz                                          Full S0 map/time series. Only used for optimal
+                                                                combination.
+echo-[echo]_desc-[PCA|ICA]_components.nii.gz                    Echo-wise PCA/ICA component weight maps.
+echo-[echo]_desc-[PCA|ICA]R2ModelPredictions_components.nii.gz  Component- and voxel-wise R2-model predictions,
+                                                                separated by echo.
+echo-[echo]_desc-[PCA|ICA]S0ModelPredictions_components.nii.gz  Component- and voxel-wise S0-model predictions,
+                                                                separated by echo.
+desc-[PCA|ICA]AveragingWeights_X.nii.gz                         Component-wise averaging weights for metric
```

Comment:
Instead of `X` should this end with `components`?

Reply:
good catch!
```suggestion
desc-[PCA|ICA]AveragingWeights_components.nii.gz                Component-wise averaging weights for metric
```